controllers/oooRequests.ts

```diff
@@ -102,4 +102,25 @@ export const updateOooRequestController = async (req: OooRequestUpdateRequest, r
         logger.error(ERROR_WHILE_UPDATING_OOO_REQUEST, err);
         return res.boom.badImplementation(ERROR_WHILE_UPDATING_OOO_REQUEST);
     }
+}
+
+export const getOooRequestsController = async (req: OooRequestCreateRequest, res: OooRequestResponse) => {
+    const { query } = req;
+    try {
+        const oooRequests = await getOooRequests(
```

Comment:
can we have query in here, and then use the value accordingly? 
For example: I've only id, then i've to pass null parameters.

Reply:
done